### DISCUSSION TITLE
Welcome to AdBlockID Discussions!

### DISCUSSION THREAD

**Discussion by realodix** [realodix/AdBlockID#678] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Recehan-Slayer** [comment]
@realodix gan itu nano adblocker yg di readme bukanya uda gnti owner dan katanya dipasangi malware

**Reply by realodix** [reply]
Buat disscussion baru aja Gan, biar rapih 🙏
